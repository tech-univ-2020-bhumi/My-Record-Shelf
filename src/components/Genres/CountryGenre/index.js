/* eslint-disable react/jsx-filename-extension */
import React from 'react';
import propTypes from 'prop-types';
import getRecord from '../../getRecords';
import useRecords from '../../../hooks/useRecords';

function CountryGenre(props) {
  const { testId } = props;
  const [records, getRecords] = useRecords();
  const countryArr = getRecord('country', records);
  console.log(countryArr);
  return (
    <div data-testid={testId}>
      <div>country</div>
      <div />
    </div>
  );
}

CountryGenre.propTypes = {
  testId: propTypes.func.isRequired,
};

export default CountryGenre;
