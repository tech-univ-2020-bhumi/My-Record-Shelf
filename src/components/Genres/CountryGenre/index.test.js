/* eslint-disable react/jsx-filename-extension */
import React from 'react';
import { render, fireEvent } from '@testing-library/react';
import CountryGenre from './index';

describe('The CountryGenre component', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<CountryGenre testId="123" />);
    expect(asFragment()).toMatchSnapshot();
  });
});
