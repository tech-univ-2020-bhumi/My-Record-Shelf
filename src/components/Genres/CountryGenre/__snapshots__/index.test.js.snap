// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The CountryGenre component should render correctly 1`] = `
<DocumentFragment>
  <div
    data-testid="123"
  >
    <div>
      country
    </div>
    <div />
  </div>
</DocumentFragment>
`;
